src/shared_modules/indexer_connector/src/indexerConnector.cpp

```diff
@@ -75,7 +72,10 @@ IndexerConnector::IndexerConnector(const nlohmann::json& config, const std::stri
     }
     nlohmann::json templateData = nlohmann::json::parse(templateFile);
 
-    // Initialize template.
+    // Initialize publisher.
+    auto selector = std::make_shared<ServerSelector>(config.at("hosts"), INTERVAL, secureCommunication);
```

Comment:
Brace initialization
```suggestion
    auto selector {std::make_shared<ServerSelector>(config.at("hosts"), INTERVAL, secureCommunication)};
```